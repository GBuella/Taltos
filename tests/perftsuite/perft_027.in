setboard 4k2r/6K1/8/8/8/8/8/8 b k - 0 1
echo 4k2r/6K1/8/8/8/8/8/8 b k - 0 1
qperft 1
qperft 2
qperft 3
qperft 4
qperft 5
qperft 6
