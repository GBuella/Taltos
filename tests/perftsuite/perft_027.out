4k2r/6K1/8/8/8/8/8/8 b k - 0 1
12
38
564
2219
37735
185867
